crates/gitbutler-cli/src/main.rs

```diff
@@ -35,7 +35,7 @@ fn main() -> Result<()> {
                 Some(vbranch::SubCommands::SetBase {
                     short_tracking_branch_name,
                 }) => command::vbranch::set_base(project, short_tracking_branch_name),
-                Some(vbranch::SubCommands::ListLocal) => command::vbranch::list_local(project),
+                Some(vbranch::SubCommands::ListLocal) => command::vbranch::list_all(project),
```

Comment:
It would be reasonable to deem this out of scope, but we should add a TODO here because the command name is clearly wrong

Reply:
Good point. I need to change this line since I remove `list_local` function. I'll update the enum to `SubCommand:List`, and update the comment on the definition.